verify-certs/verify.go

```diff
@@ -0,0 +1,100 @@
+package main
```

Comment:
I find the python stuff for dealing with certificates to be pretty icky, so I wrote this in go.